src/main/java/seedu/toluist/dispatcher/CommandDispatcher.java

```diff
@@ -43,21 +45,41 @@
     private final EventsCenter eventsCenter = EventsCenter.getInstance();
     private final AliasTable aliasConfig = Config.getInstance().getAliasTable();
 
+    /**
+     * ArrayList to store previous commands entered since starting the application
+     */
+    private ArrayList<String> commandHistory;
+    private Integer historyPointer = 0;
```

Comment:
its not really used now, but later for navigating through history commands, will pass it through to a controller. want it to be updated when i pass it in